src/Form/Form.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import Input from '../Input/Input';
+import Checkbox from '../Checkbox/Checkbox';
+import '../style.css';
+
+class Form extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      first: '',
+      last: '',
+      password: '',
+      checked: true
+    }
+  }
+
+  handleEmailChange = (event) => {
+    this.setState({email: event.target.value});
+  }
+
+  handleFirstChange = (event) => {
+    this.setState({first: event.target.value});
+  }
+
+  handleLastChange = (event) => {
+    this.setState({last: event.target.value});
+  }
+
+  handlePassChange = (event) => {
+    this.setState({password: event.target.value});
+  }
+
+  handleCheckChange = (event) => {
+    if (this.state.checked) {
```

Comment:
Pretty weird way to handle this (and creative, it works indeed). Let's make a checkbox that is properly controlled instead :D